docs/fluvio/troubleshooting.mdx

```diff
@@ -1,5 +1,31 @@
 ---
 sidebar_position: 100
-title: "Troubleshooting"
-description: "Using different Fluvio versions with Fluvio Version Manager (FVM)"
+title: "Troubleshooting a Local Cluster"
+description: "Troubleshoot common issues with your local Fluvio cluster"
 ---
+
+# Troubleshooting a Local Cluster
+
+This document will help you troubleshoot common issues with your local Fluvio cluster.
+
+
+## Error: the given record is larger than the buffer max_size (16384 bytes).
+
+This error occurs when the record size is larger than the buffer size. To resolve this issue, you can either increase the producer batch size or reduce the record size by enabling a compression algorithm.
+
+To increase the producer batch size, you can add the following configuration to your connector configuration file:
+
+```yaml
+apiVersion: 0.1.0
+
+meta:
+  producer:
+    batch-size: 44.0 MB
+```
+
+
+## Where is the Fluvio logs directory?
```

Comment:
Move this up and call it "Troubleshooting a Local Cluster" - and talk about log files
 - 2 log file (SC & SPU) 
 - where do I find the them?
 - who I change log level?
 - how do I move the log file with environment variable.
 - ... etc.